src/components/Tabs/IdentificationTab.jsx

```diff
@@ -55,675 +62,758 @@ const IdentificationTab = ({
     )
   );
 
-  return (
-    <div>
-      <Paper style={paperClass}>
-        <QuestionText>
-          <I18n>
-            <En>What is the dataset title? Required in English and French.</En>
-            <Fr>
-              Quel est le titre du jeu de données? Obligatoire dans les deux
-              langues.
-            </Fr>
-          </I18n>
-          <RequiredMark passes={validateField(record, "title")} />
-          <SupplementalText>
+  async function handleGenerateDOI(){
+
+    setLoading(true);
+    
+    try {
+      await createDraftDoi(record).then(
+          response => {
+            return response.data.data.attributes
+          }
+      ).then(
+          (attributes) => {
+            setNewDraftDoi(attributes.doi);
+            updateRecord("datasetIdentifier")(attributes.doi);
+            setDoiGenerated(true);
+          }
+      ).finally(() => { 
+        setLoading(false);
+      });
+    } catch (err) {
+      // eslint-disable-next-line no-console
+      console.log(err)
+      setDoiErrorFlag(true)
+    }
+  }
+
+  async function handleCancelDOI() {
+    try{
+
+      deleteDraftDoi(newDraftDoi).then(
+          response => response.data
+      ).then(
+          statusCode => {
+            if (statusCode === 204) {
+              setDoiGenerated(false)
+              updateRecord("datasetIdentifier")("10.0000/0000")
+            } else {
+              setDoiErrorFlag(true)
+            }
+          }
+      )
+    } catch (err){
+      // eslint-disable-next-line no-console
+      console.log(err)
+      setDoiErrorFlag(true)
+    }
+  }
+
+    return (
+      <div>
+        <Paper style={paperClass}>
+          <QuestionText>
             <I18n>
               <En>
-                <p>Recommended title includes: What, Where, When.</p>
-                <p>
-                  Title should be precise enough so that the user will not have
-                  to open the dataset to understand its contents. Title should
-                  not have acronyms, special characters, or use specialized
-                  nomenclature. This will appear as the title that is shown for
-                  this dataset in the {regionInfo.catalogueTitle.en}.
-                </p>
+                What is the dataset title? Required in English and French.
```

Comment:
I don't know why there's so much space below these tags, I don't see it in my editor.

Reply:
it seems like my editor did some formatting and added a bunch of white space

Reply:
do you have the eslint plugin installed in vscode? it should read the .eslintrc file and match the formatting 🤞